packages/language-server/src/server.ts

```diff
@@ -260,23 +259,31 @@ connection.onDidChangeConfiguration((change) => {
   if (hasConfigurationCapability) {
     getLogger().trace("Reset all cached document settings");
     clearSettings();
+    // intentionally, ignoring the results of this promise
+    // and avoiding using an async function as the API of the changeHandler is `synced`
+    // Note the change object appears to be empty of VSCode which is why we attempt to obtain
```

Comment:
Please check if after changing the scope to `window` you still need this call. Maybe it will be sent on the event.

Reply:
Still not sent on the event after fixing the scope.
- https://github.com/SAP/ui5-language-assistant/commit/a47abd00c72a16a7cce960ddce4ad0a400c4c236

Reply:
Try sending the section in `onInitializaed`
```typescript
    connection.client.register(
      DidChangeConfigurationNotification.type,
      {section: "UI5LanguageAssistant"}
    );
```
I checked it and it seems to be sending the configuration when I send it

Reply:
Thanks.
- https://github.com/SAP/ui5-language-assistant/commit/0eaa2791e4316e5aaa2f264683f1124d3f85e3de